### PR TITLE
Removes Unused Turntable Code

### PR DESCRIPTION
**Pull request by Fox-McCloud**
The turntables and lasers that go with them are super old, unused code--it's also kinda scary and does things like `in world` searches.

😱 

Either case, none of this code is even used anymore, and is *barely* known about.

no CL as this is backend.

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
`old code uses qdel instead of del`

It didn't, until I went through the entire codebase hunting for `del`'s.

**Comment by taukausanake** [comment]
Another music box? What's the story behind this one?

**Comment by Spacemanspark** [comment]
Oh, this...

Paradise used to have a disco room with this in it. I miss it dearly.

**Comment by BakKeener** [comment]
Turntable sounds awesome actually, would be cool to see it make a return but with better code.

File: code/game/machinery/turntable.dm [hunk 1]
```diff
@@ -1,280 +0,0 @@
...
-/obj/effect/turntable_laser
-	name = "laser"
-	desc = "A laser..."
-	icon = 'icons/effects/lasers2.dmi'
-	icon_state = "laserred1"
-	anchored = 1
-	layer = 4
-
-/obj/item/lasermachine/New()
-	..()
```
Comment (AffectedArc07) [inline code/game/machinery/turntable.dm:172]: What

File: code/game/machinery/turntable.dm [hunk 2]
```diff
@@ -1,280 +0,0 @@
...
-	icon_state = "turntable"
-	var/playing = 0
-	anchored = 1
-
-/obj/machinery/party/mixer
-	name = "mixer"
-	desc = "A mixing board for mixing music"
-	icon = 'icons/effects/lasers2.dmi'
-	icon_state = "mixer"
-	anchored = 1
```
Comment (AffectedArc07) [inline code/game/machinery/turntable.dm:21]: This mixer upsets me from how unrealistic it is. Good riddance.

File: code/game/machinery/turntable.dm [hunk 3]
```diff
@@ -1,280 +0,0 @@
...
-					return
-				cycle++
-				if(cycle > 3)
-					cycle = 1
-				X++
-
-
-
-/obj/machinery/party/lasermachine/proc/turnoff()
-	var/area/A = src.loc.loc
```
Comment (AffectedArc07) [inline code/game/machinery/turntable.dm:278]: `src.loc.loc`

What??!?!

Comment (Fox-McCloud) [reply code/game/machinery/turntable.dm:278]: yeah....`loc.loc` is usually the area.

loc of a mob is usually a turf, and the loc of the turf is an area.

I'm sure you can break this somehow, pretty easily though.

**Pull request merged**